gradle.properties

```diff
@@ -29,6 +29,7 @@ KOTLIN_TEST_VERSION=3.3.3
 KOTLIN_VERSION=1.3.50
 KOTLINX_COLLECTIONS_IMMUTABLE_VERSION=0.1
 KOTLINX_COROUTINES_VERSION=1.3.3
+KOTLINX_VALIDATOR_VERSION = "0.2.2"
```

Comment:
```suggestion
KOTLINX_VALIDATOR_VERSION=0.2.2
```

Reply:
Oops! thanks @rachelcarmena .. I just copy paste it

Reply:
No worries!! :raised_hands: